portfolio.css

```diff
@@ -113,3 +113,8 @@ tbody,thead {
     
 }
 
+
+/*under-line*/
+.under {
```

Comment:
instead of creating a class to underline the h2 tags
why not select all h2 tags and add a property? 
```CSS
h2 {
    text-decoration: underline;
}
```
then you do not have to change the HTML, just change the CSS file

Reply:
Thanks for using text-decoration,underline's width feels good.